### PR TITLE
error handling improvements

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: apps/remix-ide/src/app/files/fileManager.ts [hunk 1]
```diff
@@ -704,7 +704,7 @@ class FileManager extends Plugin {
   }
 
   async openFile(file?: string) {
-    if (!file) {
+    if (!file && file != '') {
```
Comment (yann300) [inline apps/remix-ide/src/app/files/fileManager.ts:707]: are you really sure it's needed? or why did you need to do that?

Comment (LianaHus) [reply apps/remix-ide/src/app/files/fileManager.ts:707]: there was a case of empty line instead of the name while I was debugging

Comment (yann300) [reply apps/remix-ide/src/app/files/fileManager.ts:707]: can you check if you still  really need this? afaik `if (!file)` should do the same.
